dora/core/common/src/test/java/alluxio/UfsUrlTest.java

```diff
@@ -82,25 +99,26 @@ public void UfsUrlCtor() {
   public void basicHdfsUri() {
     UfsUrl ufsUrl = UfsUrl.createInstance("hdfs://localhost:8020/xy z/a b c");
 
-    assertTrue(ufsUrl.getAuthority().isPresent());
-    assertEquals("localhost:8020", ufsUrl.getAuthority().get().toString());
-    assertTrue(ufsUrl.getAuthority().get() instanceof SingleMasterAuthority);
-    SingleMasterAuthority authority = (SingleMasterAuthority) ufsUrl.getAuthority().get();
+    assertNotNull(ufsUrl.getAuthority());
+    assertEquals("localhost:8020", ufsUrl.getAuthority().toString());
+    assertTrue(ufsUrl.getAuthority() instanceof SingleMasterAuthority);
+    SingleMasterAuthority authority = (SingleMasterAuthority) ufsUrl.getAuthority();
     assertEquals("localhost", authority.getHost());
     assertEquals(8020, authority.getPort());
 
     assertEquals(2, ufsUrl.getDepth());
     assertEquals("a b c", ufsUrl.getName());
-    assertEquals("hdfs://localhost:8020/xy z", ufsUrl.getParentURL().toString());
-    assertEquals("hdfs://localhost:8020/", ufsUrl.getParentURL().getParentURL().toString());
+    assertEquals("hdfs://localhost:8020/xy z", ufsUrl.getParentURL().get().toString());
+    assertEquals("hdfs://localhost:8020/",
+        ufsUrl.getParentURL().get().getParentURL().get().toString());
     assertEquals("/xy z/a b c", ufsUrl.getFullPath());
     assertEquals("hdfs", ufsUrl.getScheme());
     assertEquals("hdfs://localhost:8020/xy z/a b c/d", ufsUrl.join("/d").toString());
     assertEquals("hdfs://localhost:8020/xy z/a b c", ufsUrl.toString());
   }
 
   @Test
-  public void hashCodeTest() {
+  public void hashCodeTestOfUfsUrl() {
```

Comment:
```suggestion
  public void hashCode() {
```

Reply:
"hashCode" conflicts with the java.lang.Object.
![image](https://github.com/Alluxio/alluxio/assets/30886176/47ddaad7-0b37-4764-a5fe-9aeea4724bba)